mds/locale/en_US/LC_MESSAGES/django.po

```diff
@@ -8,136 +8,217 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-01-22 13:22+0000\n"
+"POT-Creation-Date: 2019-02-28 14:43+0000\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
 "Language: en_US\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
+#: mds/access_control/authenticate.py:57
```

Comment:
please generate PO files using the `--no-location` option. Feel free to add some doc on i18n (maybe in a separated PR).